test/appengine/test_gae_manager.py

```diff
@@ -48,43 +48,38 @@ def setUp(self):
 
     def test_exceptions(self):
         # DeadlineExceededError -> TimeoutError
-        self.assertRaises(
-            urllib3.exceptions.TimeoutError,
-            self.pool.request,
-            'GET',
-            '/sleep?seconds=0.005',
-            timeout=0.001)
+        with pytest.raises(
```

Comment:
You have a number of `pytest.raises` calls that include a newline, but you never need one. Usually the call below is not needed either. So you could simply write:

```python3
with pytest.raises(urllib3.exceptions.TimeoutError):
    self.pool.request('GET', '/sleep?seconds=0.005', timeout=0.001)
```

Use `grep 'pytest.raises($' test/*/*.py` or `rg 'pytest.raises\($'` to find them, they're mostly in test_gae_manager.py, but there's also one in test_socks.py.